### 4. In the US, the estimated probability that a woman over the age of 35 dies of lung cancer equals 0.001304 for current smokers and 0.000121 for non smokers. 

##### Calculate and interpret the difference between these proportions, and compute the relative risk of cancer. Which statistics is more informative for these data? Why?

difference of proportions = $p_{1} - p_{2}$

sample relative risk = $\frac{p_{1}}{p_{2}}$

In [3]:
p_1<-0.001304
p_2<-0.000121
difference_of_prop<- p_1-p_2
difference_of_prop
relative_risk<-p_1/p_2
relative_risk
0.000121*relative_risk

[1] 0.001183

[1] 10.77686

[1] 0.001304

The difference between these proportions show that  a woman over the age of 35 dying of lung cancer is approximately .0012 greater when she is a smoker than when she is not. The sample difference of proportions show that the response (whether or not a woman over the age of 35 dies of lung cancer) depends on whether or not she is a smoker since $p_{1}$ $\ne$ $p_{2}$. That is, when the response is not independent of the group. 

The relative risk is more informative because it shows the (striking) magnitude such that woman over the age of 35 who is a current smoker dying of lung cancer is apprixately 10.78 times that of a woman over the age of 35 who is not a current smoker. 

##### Calculate and interpret the odds ratio. Explain why the relative risk and the odds ratio take similar values.

Odds ratio = Relative Risk x $\frac{1-p_{2}}{1-p_{1}}$

In [5]:
odds_ratio<-relative_risk*((1-p_2)/(1-p_1))
odds_ratio

[1] 10.78963

The odds ratio is approximately 10.79, which means that the odds of a woman over the age of 35 who is a current smoker dying of lung cancer is 10.79 times higher than one who is a non-smoker. 

The relative risk and the OR take similar values because $p_{1}$ and $p_{2}$ are both close to 0 and therefore the fraction used in the calculations for the OR equals approximately 1. So essentially, OR = relative risk* (1)= relative risk. 